KsApi/models/graphql/adapters/Project+ProjectFragment.swift

```diff
@@ -14,6 +14,7 @@ extension Project {
     rewards: [Reward] = [],
     addOns: [Reward]? = nil,
     backing: Backing? = nil,
+    isPledgeOverTimeAllowed: Bool = false,
```

Comment:
I'm confused by this - why are we setting this to false instead of getting it from the project fragment?